prepare.yml

```diff
@@ -186,7 +186,7 @@
       tags: ndt-e2e
       environment:
         http_proxy: "{{ http_proxy }}"
-      raw: "git clone {{ ndt_e2e_client_git }} {{ ndt_e2e_client_dir }}"
+      raw: "git clone --branch master {{ ndt_e2e_client_git }} {{ ndt_e2e_client_dir }}"
```

Comment:
I would be more interested in finding out the root cause of that error and eliminating it rather than hardcoding the branch in the clone command, but if this fixed things for you, and doesn't break things for others, then I suppose there is no real harm in it.

LGTM.